exasol/slc_ci_setup/templates/slc_ci_build_slc.yml.tmpl

```diff
@@ -0,0 +1,94 @@
+{% include "heading.tmpl" %}
+{% raw %}
+name: SLC-CI-Build-Flavor
+
+on:
+  workflow_call:
+    inputs:
+      flavor:
+          description: 'A flavor'
+          required: true
+          type: string
+    secrets:
+      DOCKER_USERNAME:
+        required: true
+      DOCKER_PASSWORD:
+        required: true
+
+jobs:
+
+  detect-runner:
+    runs-on: ubuntu-22.04
+    steps:
+     - uses: actions/checkout@v4
+       with:
+          submodules: 'recursive'
+
+     - name: Setup Python & Poetry Environment
+       uses: exasol/python-toolbox/.github/actions/python-environment@1.2.0
+       with:
+         python-version: '3.10'
+         poetry-version: '2.1.2'
+     - name: Detect runner
+       id: detect-runner
+       run: poetry run -- exaslc-ci get-build-runner --flavor "$FLAVOR" --github-output-var runner
+       env:
+         FLAVOR: ${{ inputs.flavor }}
+    outputs:
+     runner: ${{ steps.detect-runner.outputs.runner }}
+  build:
+    needs: detect-runner
+    runs-on:
+      labels: ${{needs.detect-runner.outputs.runner}}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+            submodules: 'recursive'
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.2.0
+        with:
+          python-version: '3.10'
+          poetry-version: '2.1.2'
+
+      - name: Build SLC ${{ inputs.flavor }}
+        id: build-slc
+        run: |
+          mkdir -p ".build_output/jobs" ".build_output/security_scan" ".build_output/metadata"
```

Comment:
Hm, we could almost move this to python in SLC ci

Reply:
Created #110 and https://github.com/exasol/script-languages-container-ci/issues/95